src/cwv/cpc-value-resolver.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+const CPC_VALUE = 1.5;
```

Comment:
https://wiki.corp.adobe.com/display/AEMSites/Success+Studio+Projected+Business+Impact+Metrics

this wiki says:
> Default to $1 CPC

I think we could consider skipping sending "Project traffic value" in the response for now.
Since, CPC is going to be set anyway from a different API or a default value of 1$ assumed, UI could do a simple calculation of "Projected Traffic Lost" * CPC to show "Projected Traffic Value".


In brief, I suggest, you send in API response only "Projected Traffic Lost" and NOT "Projected Traffic value" (for now)

Reply:
Done. `projectedTrafficValue` has been removed.